app/src/main/java/com/example/habitsmasher/ui/history/HabitEventViewFragment.java

```diff
@@ -1,22 +1,25 @@
 package com.example.habitsmasher.ui.history;
-
 import android.os.Bundle;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ImageView;
 import android.widget.TextView;
-
 import androidx.appcompat.app.AppCompatActivity;
 import androidx.fragment.app.Fragment;
 
+import com.example.habitsmasher.Habit;
 import com.example.habitsmasher.HabitEvent;
+import com.example.habitsmasher.ImageDatabaseHelper;
 import com.example.habitsmasher.R;
-
 import java.text.SimpleDateFormat;
 
 public class HabitEventViewFragment extends Fragment {
     // Habit event being displayed
     private HabitEvent _habitEvent;
+    private Habit _parentHabit;
+    private String _userId;
+    ImageView _eventImageView;
```

Comment:
minor: `private`/`public`?

Reply:
private lol, will fix that in the next PR